modules/ztp-generating-ran-policies.adoc

```diff
@@ -1,6 +1,6 @@
 // Module included in the following assemblies:
 //
-// scalability_and_performance/ztp-deploying-disconnected.adoc
+// scalability_and_performance/ztp-far-edge/ztp-deploying-far-edge-clusters-at-scale.adoc
```

Comment:
This section describes generating RAN policies using the ACM Policy Generator.  It also mentions PolicyGenTemplate which is the ZTP policy generator resource.  I believe this needs more clarification.

Reply:
Based on slack discussion, have removed this topic altogether. It is superseded by modules/ztp-generating-install-and-config-crs-manually.adoc